plumbing/format/packfile/scanner.go

```diff
@@ -138,14 +138,52 @@ func (s *Scanner) readCount() (uint32, error) {
 	return binary.ReadUint32(s.r)
 }
 
+// SeekObjectHeader seeks to specified offset and returns the ObjectHeader
+// for the next object in the reader
+func (s *Scanner) SeekObjectHeader(offset int64) (*ObjectHeader, error) {
```

Comment:
Can you add a test for this function?

Reply:
On a second thought, it is already tested indirectly through `Packfile.GetByOffset` in `TestGetByOffset`. I can add some additional tests, but it wouldn't improve the code coverage.

Reply:
@filipnavara We prefer to add tests for public functions even when they are tested indirectly.

Reply:
I agree that's a good practice. There are already missing tests for `SeekFromStart`, which confused me.